torch/_inductor/lowering.py

```diff
@@ -1797,7 +1797,6 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten.fractional_max_pool3d_backward)
 make_fallback(aten._linalg_check_errors)
 make_fallback(aten.max_pool3d_with_indices_backward)
-make_fallback(aten.multilabel_margin_loss_backward)
```

Comment:
Todo: See my PR comment next to DispatchKey.Autograd - not sure it can work like that.